namerd/storage/k8s/src/main/scala/io/buoyant/namerd/storage/K8sDtabStore.scala

```diff
@@ -53,7 +53,15 @@ class K8sDtabStore(client: Http.Client, dst: String, namespace: String)
   }
 
   private[this]type NsMap = Map[String, VersionedDtab]
-  private[this] val dtabListToNsMap: DtabList => NsMap = _.items.map(toDtabMap)(breakOut)
+  private[this] val dtabListToNsMap: Option[DtabList] => NsMap = {
```

Comment:
Yes, the changes in 5bea97b65964b5b301f91ffbb99642741bef743a (which are necessary to make one of the tests for `ServiceNamer` pass) broke this.